src/utils/pst.ts

```diff
@@ -0,0 +1,80 @@
+import axios from "axios";
+
+// ArDrive Profit Sharing Community Smart Contract
+const vertoCacheURL =
```

Comment:
Consider renaming to `cachedContractUrl`. As written, it seems like a URL to a generalized Verto cache or some sort of cached Verto object.

Reply:
[Updated](https://github.com/th8ta/ArConnect/pull/33/commits/f76faa99ec9bfb48f340cbe9b4857e805e4e2582)